lib/common/formats.js

```diff
@@ -55,9 +55,20 @@ function formattedVariables(format, properties) {
       break;
   }
 
-  return properties.map(function(prop) {
+  return dictionary.allProperties.map(function(prop) {
       var to_ret_prop = prefix + prop.name + ': ';
-      to_ret_prop += (prop.attributes.category==='asset' ? '"'+prop.value+'"' : prop.value);
+      let value = prop.value;
+
+      if (useVariables && dictionary.usesReference(prop.original.value)) {
+        var ref = dictionary.getReference(prop.original.value).name;
+        if (format === 'css') {
+          value = `var(${prefix}${ref})`
```

Comment:
nitpicking here, but in the output it generates something like:
```
:root {
  --token-color-error: var(  --token-color-generic-red);
}
```
and that extra space inside the var() is... annoying :)

I think more correctly the `prefix` value at line 46 should be `--` and not `  --` (formally, more correct) and then we should add two extra spaces in output when we generate the line (`var to_ret_prop = prefix + prop.name + ': ';`)